benchmark/blankenbach/case2/ReadMe.txt

```diff
@@ -0,0 +1,24 @@
+* case2a:
```

Comment:
The Blankenbach stuff is still somewhat incomplete. Maybe we should do this in a separate PR.